tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -29,10 +29,23 @@ arp/test_arp_dualtor.py::test_proxy_arp_for_standby_neighbor:
 #######################################
 bfd/test_bfd.py:
   skip:
-    reason: "Test not supported for 201911 images or older, other than mlnx4600c and cosco-8102. Skipping the test"
+    reason: "Test not supported for 201911 images or older, other than mlnx4600c and cisco-8102. Skipping the test"
     conditions:
       - "(release in ['201811', '201911']) or (platform not in ['x86_64-mlnx_msn4600c-r0', 'x86_64-8102_64h_o-r0'])"
 
+bfd/test_bfd.py::test_bfd_basic:
+  skip:
+    reason: "Test not supported for cisco-8102 as it doesnt support single hop BFD. Skipping the test"
+    conditions:
+      - "platform not in ['x86_64-8102_64h_o-r0']"
```

Comment:
Hi @siqbal1986 @prsunny , the skip condition doesn't seem right here. It should be "platform in ['x86_64-8102_64h_o-r0']" to skip the test on cisco-8102. Could you help confirm? Thanks.